#Controller State
uint8 CONTROLLER_STATE_LAND = 0
uint8 CONTROLLER_STATE_TAKEOFF=1
uint8 CONTROLLER_STATE_HOLD = 2
uint8 CONTROLLER_STATE_MISSION_FOLLOW=3

uint8 controller_state
bool controller_ready
uint8 mission_status
